osu.Game.Rulesets.Osu/Mods/OsuModRandom.cs

```diff
@@ -24,9 +27,23 @@ public class OsuModRandom : ModRandom, IApplicableToBeatmap
         private const byte border_distance_x = 128;
         private const byte border_distance_y = 96;
 
+        [SettingSource("Seed", "Seed for the random number generator")]
+        public Bindable<string> Seed { get; } = new Bindable<string>
```

Comment:
Why not make this an int with a large selectable value? I think the UX is better if the user can drag on a slider to change the seed rather than typing a number. Especially since they won't know that it's limited to integer input until they try and fail.

Reply:
Wouldn't an `OsuNumberBox` like in the sample implementation here #12602 be more suited for this?